214: grammar correction notebook (#839)
openvinotoolkit/openvino_notebooks: notebooks/214-grammar-correction/README.md

# Grammatical Error Correction with OpenVINO

> Grammatical Error Correction (GEC) is the task of correcting different kinds of errors in text such as spelling, punctuation, grammatical, and word choice errors. 

Reviewer: ```suggestion
Grammatical Error Correction (GEC) is the task of correcting different types of errors in text such as spelling, punctuation, grammatical and word choice errors. 
```

> GEC is typically formulated as a sentence correction task. A GEC system takes a potentially erroneous sentence as input and is expected to transform it into its corrected version. See the example given below: 

Reviewer: ```suggestion
GEC is typically formulated as a sentence correction task. A GEC system takes a potentially erroneous sentence as input and is expected to transform it into a more correct version. See the example given below:
```

| Input (Erroneous)                                         | Output (Corrected)                                       |
| --------------------------------------------------------- | -------------------------------------------------------- |

> |She see Tom is catched by policeman in park at last night. | She saw Tom caught by a policeman in the park last night.|

Reviewer: ```suggestion
| I like to rides my bicycle. | I like to ride my bicycle. |
```

Reviewer: If you have other examples that you think are better, please feel free to suggest a better one. @eaidova

This tutorial shows how to perform grammatical error correction using OpenVINO. We will use pre-trained models from the [Hugging Face Transformers](https://huggingface.co/docs/transformers/index) library. To simplify the user experience, the [Hugging Face Optimum](https://huggingface.co/docs/optimum) library is used to convert the models to OpenVINO™ IR format.

## Notebook Contents

The tutorial consists of the following steps:

- Install prerequisites

> - Download and convert models from a public source using the OpenVINO integration with Hugging Face Optimum.

Reviewer: ```suggestion
- Download and convert models from a public source using the [OpenVINO integration with Hugging Face Optimum](https://huggingface.co/blog/openvino).
```
